bitcoin/src/consensus/encode.rs

```diff
@@ -936,10 +936,20 @@ mod tests {
         assert_eq!(serialize(&cd), vec![5, 0, 0, 0, 162, 107, 175, 90, 1, 2, 3, 4, 5]);
     }
 
+    fn prefix_varint_len(data: &Vec<u8>) -> Vec<u8> {
```

Comment:
I wonder why clippy doesn't complain about accepting `&Vec<u8>`, IIRC it did some time ago.